melos.yaml

```diff
@@ -6,9 +6,4 @@ packages:
 
 command:
   version:
-    # Only allow versioning to happen on main branch.
-    branch: main
-    # Generate commit links in package changelogs.
-    linkToCommits: true
-    # Additionally build a changelog at the root of the workspace.
-    workspaceChangelog: true
```

Comment:
- Removed because I want to use the version command outside of the main branch.
- Remove `linkToCommits: true` because it is the default.
- If the package has CHANGELOG.md, `workspaceChangelog` is `false` because it is not needed.